### PR TITLE
Docs: adding the PbC label to demos.

### PR DESCRIPTION
**Pull request by godai78**
### Suggested merge commit message ([convention](https://ckeditor.com/docs/ckeditor5/latest/framework/contributing/git-commit-message-convention.html))

Docs: adding the "Powered by CKBox" label to feature guides demos. Closes cksource/ckeditor5-commercial#5826

---

### Additional information

Includes:

- [x] PbC label added to CKBox in all demos in feature guides that have CKBox service attached
- [x] additional `insertImage` instances added where lacked
- [x] various minor fixes in the snippets where deemed applicable

_Merge together with https://github.com/cksource/ckeditor5-commercial/pull/5837_

### CODE REVIEW

File: packages/ckeditor5-list/docs/_snippets/features/todo-list.js [hunk 1]
```diff
@@ -32,6 +32,9 @@ ClassicEditor
 				'toggleImageCaption', 'imageTextAlternative', 'ckboxImageEdit'
 			]
 		},
+		ckbox: {
+			forceDemoLabel: true
+		},
```
Comment (mmotyczynska) [inline packages/ckeditor5-list/docs/_snippets/features/todo-list.js:37]: For some reason I don't see `insert with file manager` button in this demo.
image

File: packages/ckeditor5-ui/docs/_snippets/features/blocktoolbar.js [hunk 2]
```diff
@@ -125,6 +125,9 @@ BalloonEditor
 		toolbar: {
 			items: [ 'bold', 'italic', '|', 'link' ]
 		},
+		ckbox: {
+			forceDemoLabel: true
+		},
```
Comment (mmotyczynska) [inline packages/ckeditor5-ui/docs/_snippets/features/blocktoolbar.js:130]: In this demo I don't see the `insert with file manager` toolbar button.

**Review by mmotyczynska — COMMENTED** [review]
Generally looks OK 👍  I added a few comments, please check if they are relevant 🙂

File: packages/ckeditor5-image/docs/_snippets/features/image-insert-via-url.js [hunk 3]
```diff
@@ -23,6 +23,9 @@ ClassicEditor
 				'|', 'bulletedList', 'numberedList', 'outdent', 'indent'
 			]
 		},
+		ckbox: {
+			forceDemoLabel: true
+		},
```
[the remark this answers is not in the document]
Comment (godai78) [reply packages/ckeditor5-image/docs/_snippets/features/image-insert-via-url.js:28]: This one does not need it to not draw attention away.

File: docs/_snippets/features/mathtype.js [hunk 4]
```diff
@@ -9,7 +9,7 @@ import { ClassicEditor } from '@ckeditor/ckeditor5-editor-classic';
 import { CloudServices } from '@ckeditor/ckeditor5-cloud-services';
 import { CS_CONFIG } from '@ckeditor/ckeditor5-cloud-services/tests/_utils/cloud-services-config';
 import ArticlePluginSet from '@ckeditor/ckeditor5-core/tests/_utils/articlepluginset';
-import { EasyImage } from '@ckeditor/ckeditor5-easy-image';
+// import { EasyImage } from '@ckeditor/ckeditor5-easy-image';
```
[the remark this answers is not in the document]
Comment (godai78) [reply docs/_snippets/features/mathtype.js:12]: In commit.

File: docs/_snippets/features/toolbar-nested-label.js [hunk 5]
```diff
@@ -29,6 +29,9 @@ ClassicEditor
 				'|', 'toggleImageCaption', 'imageTextAlternative', '|', 'ckboxImageEdit' ]
 		},
 		cloudServices: CS_CONFIG,
+		ckbox: {
+			forceDemoLabel: true
+		},
```
[the remark this answers is not in the document]
Comment (godai78) [reply docs/_snippets/features/toolbar-nested-label.js:34]: In commit.

File: docs/_snippets/features/toolbar-nested-simple.js [hunk 6]
```diff
@@ -36,6 +36,9 @@ ClassicEditor
 				'|', 'toggleImageCaption', 'imageTextAlternative', '|', 'ckboxImageEdit' ]
 		},
 		cloudServices: CS_CONFIG,
+		ckbox: {
+			forceDemoLabel: true
+		},
```
[the remark this answers is not in the document]
Comment (godai78) [reply docs/_snippets/features/toolbar-nested-simple.js:41]: In commit.

File: docs/_snippets/features/ui-language-content.js [hunk 7]
```diff
@@ -28,6 +28,9 @@ ClassicEditor
 				'toggleImageCaption', 'imageTextAlternative', 'ckboxImageEdit'
 			]
 		},
+		ckbox: {
+			forceDemoLabel: true
+		},
```
[the remark this answers is not in the document]
Comment (godai78) [reply docs/_snippets/features/ui-language-content.js:33]: This one looks like a translation error. @pomek - can we address it?

File: packages/ckeditor5-list/docs/_snippets/features/todo-list.js (hunk 1 above)
Comment (godai78) [reply packages/ckeditor5-list/docs/_snippets/features/todo-list.js:37]: @niegowski - can you please have a look at it in your spare time? The imports are there, just don't work.  
  
I don't find this a bloker for merging.

File: packages/ckeditor5-ui/docs/_snippets/features/blocktoolbar.js (hunk 2 above)
Comment (godai78) [reply packages/ckeditor5-ui/docs/_snippets/features/blocktoolbar.js:130]: In commit.

File: packages/ckeditor5-list/docs/_snippets/features/todo-list.js (hunk 1 above)
Comment (godai78) [reply packages/ckeditor5-list/docs/_snippets/features/todo-list.js:37]: Further observations: looks like CKBox is imported, but does not work anyway - adding it to the toolbar returned an error:

Comment (godai78) [reply packages/ckeditor5-list/docs/_snippets/features/todo-list.js:37]: Some problems with the editing, too. I comment the service out until we can identify and fix the problem.

File: docs/_snippets/features/ui-language-content.js (hunk 7 above)
[the remark this answers is not in the document]
Comment (godai78) [reply docs/_snippets/features/ui-language-content.js:33]: I have verified these errors have nothing to do with the PbC label, so I believe we can proceed and extract this as a follow-up.

[the remark this answers is not in the document]
Comment (mmotyczynska) [reply docs/_snippets/features/ui-language-content.js:33]: Agree.

File: packages/ckeditor5-image/docs/_snippets/features/image-insert-via-url.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (mmotyczynska) [reply packages/ckeditor5-image/docs/_snippets/features/image-insert-via-url.js:28]: Then we can remove this code, right?

**Pull request merged**